src/task.js

```diff
@@ -0,0 +1,116 @@
+import completeTask from './complete.js';
+
+class TaskFunctions {
```

Comment:
Similar to pure functions, I think the class should not be acessing external functions/modules.

Reply:
I'll work on it